packages/typography/src/elements/CodeBlock.tsx

```diff
@@ -41,14 +43,20 @@ export const CodeBlock = React.forwardRef<HTMLPreElement, ICodeBlockProps>(
     const containerRef = useRef<HTMLDivElement>(null);
```

Comment:
If we have decided on `undefined` do we want to do preventative maintenance and remove `null`? Don't think this would be a breaking change.

Reply:
Removing breaks typescript for `useScrollRegion`.

```
Type 'MutableRefObject<HTMLDivElement | undefined>' is not assignable to type 'RefObject<HTMLElement>'.
  Types of property 'current' are incompatible.
    Type 'HTMLDivElement | undefined' is not assignable to type 'HTMLElement | null'.
```